### PR TITLE
update Readme, fields and actions.

### PR DESCRIPTION
**Pull request by aranajhonny**
@juancampa review this please.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# Twitter Driver
+
+An [Membrane.io](https://membrane.io/) driver for Twitter.
```
Comment (juancampa) [inline README.md]:
```suggestion
A [Membrane.io](https://membrane.io/) driver for Twitter.
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+## Usage
+
+1. Install the [Membrane VSCode Extension](https://marketplace.visualstudio.com/items?itemName=membrane.membrane).
+2. Setup on extension your [Membrane's CLI binary (mctl)](https://membrane.io/download) path.
+3. Login / Sign up with ```mctl login```.
+4. Update the program on Membrane with the VSCode Command palette `(cmd+shift+p)`\
+  ```> Membrame: Update current program```
```
Comment (juancampa) [inline README.md]: We can remove this entire part as it's going to be outdated pretty soon and it's understood that you need to be using Membrane to use this driver

File: README.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+## Actions
+
+Configure [APP KEY and APP SECRET](https://developer.twitter.com/en/portal/register/keys)
+
+$~~~~$`mctl action "twitter:configure(APP_KEY: '<string>', APP_SECRET: '<string>')"`
+
+Create a tweet (also support for polls)
```
Comment (juancampa) [inline README.md]:
```suggestion
Create a tweet (it also support polls!)
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+$~~~~$`mctl action "twitter:tweet(text: 'Hi!')"`
```
Comment (juancampa) [inline README.md]:
```suggestion
$~~~~$`mctl action "twitter:tweet(text:'Hi!')"`
```
Currently grefs can't have whitespace. We can fix that later but let's remove all whitespace for now

File: util.ts [hunk 5]
```diff
@@ -39,6 +37,42 @@ export type ResolverInfo = {
...
+    .split('&')
+    .reduce((acc, query) => {
+      const [key, value] = query.split('=');
+
+      if (key) {
+        acc[key] = decodeURIComponent(value);
+      }
+
+      return acc;
+    }, {} as Record<string, string>);
```
Comment (juancampa) [inline util.ts]:
```suggestion
export const parseQS = (qs: string): Record<string, string> =>
  Object.fromEntries(new URLSearchParams(qs).entries())
```

Something like this should work

File: util.ts [hunk 6]
```diff
@@ -1,31 +1,29 @@
...
   method: string,
   path: string,
   query?: any,
   body?: string
-) {
-  if (!state.token) {
-    throw new Error(
-      "You must first invoke the configure action with an API token"
-    );
-  }
```
Comment (juancampa) [inline util.ts]: Any reason to remove this? It's good to see this error when a user forgets to invoke `configure`

File: README.md [hunk 7]
```diff
@@ -0,0 +1,85 @@
+# Twitter Driver
+
+A [Membrane.io](https://membrane.io/) driver for Twitter.
+
+## Actions
```
Comment (juancampa) [inline README.md]:
```suggestion
## Setup
```

File: README.md [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+
+Get [Client key and secret](https://developer.twitter.com/en/portal) from twitter dev portal and setup your [Callback URI](https://developer.twitter.com/en/docs/apps/callback-urls) with the program Endpoint URL.
+
+$~~~~$`mctl action "twitter:configure(CLIENT_ID:'<string>',CLIENT_SECRET:'<string>')"`
+
+Create a tweet (it also support polls!)
+
+$~~~~$`mctl action "twitter:tweet(text:'Hi!')"`
```
Comment (juancampa) [inline README.md]: It would be nice to separate this into two examples `Create a tweet` and `Create a poll tweet`

**Review by juancampa — CHANGES_REQUESTED** [review]

**Comment by aranajhonny** [comment]
@juancampa

**Review by juancampa — APPROVED** [review]

**Pull request merged**